### PR TITLE
Pin pyzmq < 22.0.0

### PR DESCRIPTION
**Pull request by jeremyliweishih**
Testing as our windows unit tests are breaking.

### CODE REVIEW

File: docs/source/release_notes.rst [hunk 1]
```diff
@@ -8,6 +8,7 @@ Release Notes
         * Added support for ``scikit-learn`` ``v0.24.0`` :pr:`1733`
     * Fixes
         * Removed ``random_state`` arg from ``get_pipelines`` in ``AutoMLSearch`` :pr:`1719`
+        * Pinned pyzmq at less than 22.0.0 till we add support :pr:`1756`
```
Comment (angela97lin) [inline docs/source/release_notes.rst:11]: lol I don't really know if this is our issue 🤷

**Review by angela97lin — APPROVED** [review]
Yay, glad this worked! 👍

File: docs/source/release_notes.rst (hunk 1 above)
Comment (dsherry) [reply docs/source/release_notes.rst:11]: @angela97lin what would you say here?

**Review by dsherry — APPROVED** [review]
👍

**Comment by jeremyliweishih** [comment]
@dsherry @chukarsten filed #1759 to unpin this.

File: docs/source/release_notes.rst (hunk 1 above)
Comment (angela97lin) [reply docs/source/release_notes.rst:11]: @dsherry Hmmm, maybe something along the lines of "Pinned pyzmq < 22.0.0 until nbval supports pyzmq == 22.0.0"? It looks like that's where the attempt to install is happening and failing...

**Pull request merged**